src/cata_tiles.cpp

```diff
@@ -2314,7 +2314,10 @@ bool cata_tiles::draw_entity( const Creature &critter, const tripoint &p, lit_le
             std::string chosen_id = ent_name.str();
             if( m->has_effect( effect_ridden ) ) {
                 int pl_under_height = 6;
-                draw_entity_with_overlays( g->u, p, ll, pl_under_height );
+                if( m->mounted_player ) {
+                    player *pl = m->mounted_player;
```

Comment:
Seriously: what's with those local variables? The have a meaningless name ("pl" as compared to "m->mounted_player", the later at least tells what it is), and are usually used only once.

Using the `m->mounted_player` directly would make these statements much more readable.